### DISCUSSION TITLE
Hello, I want to ask, I expanded my system, then relaxed it, and then calculated the AIMD of 1300K frames at 300 K. I used this data as the initial data set and trained the model with DeePMD-kit, and found that the fitting result was very good. Do I need to add more data for training? Looking forward to your reply, thank you.

### DISCUSSION THREAD

**Discussion by gao890** [deepmodeling/deepmd-kit#3154] [Q&A]
<img width="234" alt="image" src="https://github.com/deepmodeling/deepmd-kit/assets/139831189/12632fd5-7b4e-4c0f-a2a4-c386afc85f42">
<img width="229" alt="image" src="https://github.com/deepmodeling/deepmd-kit/assets/139831189/d31538a9-6ce2-432a-a648-de7807560f40">

**Comment by Saberve** [comment]
You can calculate the RMSE of energy and force on the test set to evaluate whether it meets your accuracy requirements.
